app/frontend/src/search/data/locations.js

```diff
@@ -0,0 +1,145 @@
+export const locations = [
+    'Barking and Dagenham',
+    'Barnet',
+    'Bexley',
+    'Brent',
+    'Bromley',
+    'Camden',
+    'City of London',
+    'Croydon',
+    'Ealing',
+    'Enfield',
+    'Greenwich',
+    'Hackney',
+    'Hammersmith and Fulham',
+    'Haringey',
+    'Harrow',
+    'Havering',
+    'Hillingdon',
+    'Hounslow',
+    'Islington',
+    'Kensington and Chelsea',
+    'Kingston upon Thames',
+    'Lambeth',
+    'Lewisham',
+    'Merton',
+    'Newham',
+    'Redbridge',
+    'Richmond upon Thames',
+    'Southwark',
+    'Sutton',
+    'Tower Hamlets',
+    'Waltham Forest',
+    'Wandsworth',
+    'Westminster',
+    'Bath',
+    'Birmingham',
+    'Bradford',
+    'Brighton and Hove',
+    'Bristol',
+    'Cambridge',
+    'Canterbury',
+    'Carlisle',
+    'Chelmsford',
+    'Chester',
+    'Chichester',
+    'Coventry',
+    'Derby',
+    'Exeter',
+    'Gloucester',
+    'Hereford',
+    'Kingston-upon-Hull',
+    'Lancaster',
+    'Leeds',
+    'Leicester',
+    'Lichfield',
+    'Lincoln',
+    'Liverpool',
+    'London',
+    'Manchester',
+    'Newcastle',
+    'Norwich',
+    'Nottingham',
+    'Oxford',
+    'Peterborough',
+    'Plymouth',
+    'Portsmouth',
+    'Preston',
+    'Ripon',
+    'Salford',
+    'Salisbury',
+    'Sheffield',
+    'Southampton',
+    'St Albans',
+    'Stoke-on-Trent',
+    'Sunderland',
+    'Truro',
+    'Wakefield',
+    'Wells',
+    'Winchester',
+    'Wolverhampton',
+    'Worcester',
+    'York',
+    'Avon',
```

Comment:
Should this be two different arrays?  I had to double-take and ponder when I realised the alphabetisation re-started.